src/DetailsView/components/details-view-content.tsx

```diff
@@ -134,6 +137,18 @@ export const DetailsViewContent = NamedFC<DetailsViewContentProps>('DetailsViewC
             ),
         );
 
+        const assessmentCardsViewData = props.deps.getCardViewData(
+            props.storeState.assessmentStoreData,
```

Comment:
We should use the `assessmentStoreData` we get on line 170 (since that will appropriately switch for Quick Assess and Assessment). We should also look to mimic that pattern for the card selection data.